_ml-commons-plugin/cluster-settings.md

```diff
@@ -143,8 +143,12 @@ The default value allows you to upload a model file from any http/https/ftp/loca
 
 ### Setting
 
+The default URL value for this trusted URL setting is not secure. To ensure the security, please use you own regex string to the repostory that contains your models, for example `https://github.com/opensearch-project/ml-commons/blob/2.x/ml-algorithms/src/test/resources/org/opensearch/ml/engine/algorithms/text_embedding/*`.
+{: .warning }
+
+
 ```
-plugins.ml_commons.trusted_url_regex: ^(https?\|ftp\|file)://[-a-zA-Z0-9+&@#/%?=~_\|!:,.;]*[-a-zA-Z0-9+&@#/%=~_\|]
+plugins.ml_commons.trusted_url_regex: <model-repository-url>
```

Comment:
Default value should be `"^(https?|ftp|file)://[-a-zA-Z0-9+&@#/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|]"`

Reply:
The default value is called out below. I'm ok with switching it back, but I figured we would try and de-emphasize the default value for this setting to encourage people to change it.

Reply:
Sure, I mean the default value of line 156 is not correct. I can't comment on line 156, so just add comment here.